[mdc-auto-init] npm index.d.ts has multiple default exports

### Bug report
npm @material/auto-init/index.d.ts has multiple default exports

### Steps to reproduce
https://repl.it/repls/ElasticKnowledgeableLocations

### Actual behavior
> node_modules/@material/auto-init/index.d.ts:38:16 - error TS2528: A module cannot have multiple default exports.

### Your Environment:
| Software         | Version(s) |
| ---------------- | ---------- |
| MDC Web          |3.2.0
| Browser          |Chrome 77.0.3865.90
| Operating System |Windows 10 (1903)

### Possible solution
Remove duplicate default export from index.d.ts

Same issue here as of the last couple of days.  I had many more after digging into an outdated project and updating material design - though upgrading the `typescript` version to >3.0, and the *jquery* `types` package to the current latest resolved all but:

```bash
  [Build]: Compiling Typescript sources into JavaScript...  
node_modules/@material/auto-init/dist/mdc.autoInit.d.ts(44,20): error TS2528: A module cannot have multiple default exports.
node_modules/@material/auto-init/dist/mdc.autoInit.d.ts(45,20): error TS2528: A module cannot have multiple default exports.
node_modules/@material/auto-init/index.d.ts(38,16): error TS2528: A module cannot have multiple default exports.
node_modules/@material/auto-init/index.d.ts(39,16): error TS2528: A module cannot have multiple default exports.
```

and

```bash
  [Build]: Generating Typescript declarations...  
node_modules/@material/auto-init/dist/mdc.autoInit.d.ts(44,20): error TS2528: A module cannot have multiple default exports.
node_modules/@material/auto-init/dist/mdc.autoInit.d.ts(45,20): error TS2528: A module cannot have multiple default exports.
node_modules/@material/auto-init/index.d.ts(38,16): error TS2528: A module cannot have multiple default exports.
node_modules/@material/auto-init/index.d.ts(39,16): error TS2528: A module cannot have multiple default exports.
```

I upgraded material-components-web to `"material-components-web": "^4.0.0",`

and `"@types/material__auto-init": "^0.35.1",` seems to be the latest version (my other `@types/material...` packages are at their latest as well.

The issue is not stopping the build from happening, but to add to the reported issue...

--- a/packages/mdc-auto-init/index.ts
+++ b/packages/mdc-auto-init/index.ts
@@ -64,7 +64,7 @@ function emit<T extends object>(evtType: string, evtData: T, shouldBubble = fals
 /**
  * Auto-initializes all MDC components on a page.
  */
-export function mdcAutoInit(root = document) {
+function mdcAutoInit(root = document) {
   const components = [];
   let nodes: Element[] = [].slice.call(root.querySelectorAll(`[${AUTO_INIT_ATTR}]`));
   nodes = nodes.filter((node) => node.getAttribute(AUTO_INIT_STATE_ATTR) !== INITIALIZED_STATE);
@@ -122,3 +122,4 @@ mdcAutoInit.deregisterAll = function() {
 
 // tslint:disable-next-line:no-default-export Needed for backward compatibility with MDC Web v0.44.0 and earlier.
 export default mdcAutoInit;
+export {mdcAutoInit};

